### PR TITLE
Fixing Bounding Box format for Text Detection isn't specified

### PR DESCRIPTION
**Pull request by EuanHoll**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

To fix a ticket I made.
https://github.com/open-mmlab/mmocr/issues/1733

## Modification

Adds a description of the bounding box format for easier understanding of the dext detection dataset format.

## BC-breaking (Optional)

Nope

## Use cases (Optional)

Easier creation of custom scripts for ocr dataset creation.

## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmocr/blob/main/.github/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmocr/blob/main/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [X] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects.
- [X] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/en/migration/dataset.md [hunk 1]
```diff
@@ -122,6 +122,7 @@ The `TextDetDataset` holds the information required by the text detection task,
     ]
 }
 ```
+The bounding box format is as follows: `[min_x, min_y, width, height]`
```
Comment (gaotongxiao) [inline docs/en/migration/dataset.md:125]:
```suggestion

The bounding box format is as follows: `[min_x, min_y, max_x, max_y]`
```

**Review by gaotongxiao — COMMENTED** [review]
Thanks for your contribution! Your interpretation was wrong however :) And don't forget to sign CLA before we can get this PR merged!

**Comment by EuanHoll** [comment]
Signed, and thanks for fixing!

**Review by gaotongxiao — APPROVED** [review]

**Comment by jason102811** [comment]
Hi @EuanHoll  ！First of all, we want to express our gratitude for your significant PR in the MMDet . Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA

If you have WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request merged**